packages/lwc/html.d.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+// This file solely provides type definitions, so there is no corresponding .js file.
+import '@lwc/engine-dom/html';
```

Comment:
This is a little weird, since it breaks with our convention of "The `lwc` package just re-exports other packages," but I guess it's fine since it's less typing for the user.